docs/protocol/network/data-structures/transaction.md

```diff
@@ -0,0 +1,136 @@
+# Transaction
+
+In the Witnet network protocol, a `transaction` is formatted as follows:
+
+| Field     |    Type    | Description                                                               |
+| --------- | :--------: | ------------------------------------------------------------------------- |
+| `version` |   `u32`    | The transaction data format version number                                |
+| `inputs`  | `[input]`  | A list of 1 or more transaction inputs                                    |
+| `outputs` | `[output]` | A list of transaction outputs                                             |
+| `claims`  | `[claim]`  | A list of 1 or more claims (i.e. spending conditions), one for each input |
+
+The inputs and claims lists cannot be empty as trasactions require at least to consume an unspent output.
+
+Signature structures are defined in the [Signature] section.
+
+## Inputs (aka outpoints)
```

Comment:
It's true that all inputs are of the type `outpoint` but I wouldn't dare to assimilate them (just like I wouldn't say `Bakers (aka humans)`)

Generally speaking, `outpoint` is just an alias for a tuple of a transaction's id and the index of one of its outputs. Giving too much protagonism to the `outpoint` concept may render the documentation more confusing and less approachable for people not familiar to it.